lib/stacks/infra/core.ts

```diff
@@ -410,6 +412,29 @@ export class CoreStack extends Stack {
             projectionType: ProjectionType.ALL,
         });
 
+        // App Configuration Table
+        const appConfigTable = new Table(this, 'mlspace-ddb-app-configuration', {
```

Comment:
why are we using a DDB table rather than using something like SSM Parameters? Why do we need config versioning at all?

Reply:
1. Audting - our customers like being able to audit things and look at the config history, so this way we have a clear trail
2. Rollbacks - if someone pushes a config that does something they don't like it's very easy to rollback to the prior version
3. Metadata - using a DB makes it trivially easy to store metadata for config changes (who changed, the reason why)